tests/deployment/vlan_networks.py

```diff
@@ -215,17 +215,21 @@ def delete_networks():
                 for attachment in attachments:
                     all_attachments.append((node.label,
                                             node.nics[0].label,
-                                            attachment.network.label))
+                                            attachment.network.label,
+                                            attachment.channel))
```

Comment:
4 elements in a tuple is enough that it starts to get hard to keep track of. Suggestion: use a namedtuple.